e2e/common/config/config_test.go

```diff
@@ -120,6 +122,16 @@ func TestRunConfigExamples(t *testing.T) {
 		Eventually(IntegrationPodPhase(ns, "property-secret-route"), TestTimeoutLong).Should(Equal(corev1.PodRunning))
 		Eventually(IntegrationConditionStatus(ns, "property-secret-route", v1.IntegrationConditionReady), TestTimeoutShort).Should(Equal(corev1.ConditionTrue))
 		Eventually(IntegrationLogs(ns, "property-secret-route"), TestTimeoutShort).Should(ContainSubstring("my-secret-external-value"))
+
+		// check integration schema does not contains unwanted default trait value.
+		gvk := schema.GroupVersionKind{Group: v1.SchemeGroupVersion.Group, Version: v1.SchemeGroupVersion.Version, Kind: v1.IntegrationKind}
```

Comment:
I see this is duplicated everywhere. Create a test support function in order to avoid too much duplication and difficulty in code reading.